helm_deploy/values-prod.yaml

```diff
@@ -19,21 +19,3 @@ generic-prometheus-alerts:
   alertSeverity: digital-prison-service
   rdsAlertsDatabases:
     cloud-platform-67257a8b15c64e7f: "Whereabouts API"
-
-ingress:
```

Comment:
Didn't you just add those in the last PR?

Reply:
Added in previous PR to push old ingress changes to prod - this PR removes the old ingress 🤷